x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/empty_state.test.tsx

```diff
@@ -21,7 +23,7 @@ describe('EmptyState', () => {
 
     expect(wrapper.find('h2').text()).toEqual('Add documents to generate a Search UI');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('/reference-ui-guide.html')
+      expect.stringContaining(`${docLinks.appSearchSearchUI}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchSearchUI)
```
No need for a template literal here